src/test/java/uk/gov/companieshouse/limitedpartnershipsapi/interceoptor/UserAuthenticationInterceptorTest.java

```diff
@@ -0,0 +1,75 @@
+package uk.gov.companieshouse.limitedpartnershipsapi.interceoptor;
```

Comment:
package name interceptor spelled incorrectly

Reply:
Demutilated this word.